app/http.php

```diff
@@ -766,7 +767,7 @@ function (string $runtimeId, ?string $payload, string $path, string $method, mix
             $numericParams = ['timeout', 'cpus', 'memory'];
             foreach ($numericParams as $numericParam) {
                 if (!empty($$numericParam) && !is_numeric($$numericParam)) {
-                    $$numericParam = \intval($$numericParam);
+                    $$numericParam = \floatval($$numericParam);
```

Comment:
This applies to timeout and memory too, but those shouldn't be floats.

I think we need to remove CPUs from this part,  and split it to `intParams` and `floatParams`